release/release_tests.yaml

```diff
@@ -2473,7 +2473,7 @@
   working_dir: long_running_tests
 
   frequency: weekly
-
+  python: "3.7"
```

Comment:
remove these maybe we should be using the default version which is 3.8

Reply:
version 3.8 conflicts with app_config.yaml who sets default docker image versions to py37. See this https://github.com/ray-project/ray/blob/0bded3ba9ffe99299285e03b1488ec1ac8b26e59/release/long_running_tests/app_config.yaml#L1 

To update the docker images we need to update a lot of app_config.yaml files. Shall I do that in this PR?

Reply:
These app_config are here but not really used by BYOD. I just clean up all these files actually, if you rebase they are gone now